### DISCUSSION TITLE
Welcome to meawto.github.io Discussions!

### DISCUSSION THREAD

**Discussion by meawcafe** [meawto/meawto.github.io#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nikoscocos** [comment]
Hello! I'm from the nikitOS project and I've looked at your web OS.
It is very good.

I would like to ask if I can make a theme for my web operating system in the style of your project?
Thanks for your work.

![image](https://user-images.githubusercontent.com/77122703/209710162-67f75932-247e-48f0-8559-ab35308874af.png)

**Comment by meawcafe** [comment]
привет друг разработчик

I hope you are well. I am flattered to know that you are interested in my project. Feel free to use this and my other projects.
~ Open Source ~!

.dark-mode {
    --main-color0: #131313;
    --main-color1: #161616;
    --main-color2: #1d1d1d;
    --main-color3: #232323;
    --main-color4: #272727;
    --main-color5: #303030;
    --main-color6: #3f3f3f;
    --main-color7: #c2c2c2;
    --main-color8: #eeeeee;
    --interface-color0: rgba(0, 0, 0, 0.4);
    --interface-color1: rgba(0, 0, 0, 0.2);
    --interface-color2: rgba(255, 255, 255, 0.2);
    --interface-color3: rgba(0, 0, 0, 0.6);
    --interface-color4: rgba(19, 19, 19, 0.4);
    --interface-color5: 255, 255, 255;
    --interface-color6: rgb(28, 28, 28);
    --interface-color7: rgba(0, 0, 0, 0.8);
    --interface-color8: rgba(0, 0, 0, 0.4);
    --interface-color9: rgb(209, 209, 209);
    --interface-color10: rgba(194, 194, 194, 0.6);
    --color-name: rgb(190, 190, 190);
    --color-status: rgb(160, 160, 160);
    --color-sub-status: rgb(130, 130, 130);
}

.white-mode {
    --main-color0: #e6e6e6;
    --main-color1: #ededed;
    --main-color2: #e2e2e2;
    --main-color3: #dcdcdc;
    --main-color4: #ededed;
    --main-color5: #cfcfcf;
    --main-color6: #c0c0c0;
    --main-color7: #3d3d3d;
    --main-color8: #111111;
    --interface-color0: rgba(167, 167, 167, 0.4);
    --interface-color1: rgba(255, 255, 255, 0.2);
    --interface-color2: rgba(0, 0, 0, 0.2);
    --interface-color3: rgba(0, 0, 0, 0.4);
    --interface-color4: rgba(255, 255, 255, 0.4);
    --interface-color5: 0, 0, 0;
    --interface-color6: rgb(247, 247, 247);
    --interface-color7: rgba(0, 0, 0, 0.4);
    --interface-color8: rgba(255, 255, 255, 0.2);
    --interface-color9: rgb(0, 0, 0);
    --interface-color10: rgb(221, 221, 221);
    --color-name: rgb(65, 65, 65);
    --color-status: rgb(95, 95, 95);
    --color-sub-status: rgb(125, 125, 125);
}


De: Nikita ***@***.***>
Enviado:terça-feira, 27 de dezembro de 2022 15:56
Para: ***@***.***>
Cc:MILTON MARTINS ***@***.***>; ***@***.***>
Assunto: Re: [meawcafe/meawcafe.github.io] Welcome to meawto.github.io Discussions! (Discussion #1)


Hello! I'm from the nikitOS project and I've looked at your web OS.
It is very good.

I would like to ask if I can make a theme for my web operating system in the style of your project?
Thanks for your work.

[image]<https://user-images.githubusercontent.com/77122703/209710162-67f75932-247e-48f0-8559-ab35308874af.png>

**Reply by Nikoscocos** [reply]
Thank you so much!
